pkg/pod/pod.go

```diff
@@ -90,6 +90,12 @@ func NewBuilder(apiClient *clients.Settings, name, nsname, image string) *Builde
 func Pull(apiClient *clients.Settings, name, nsname string) (*Builder, error) {
 	glog.V(100).Infof("Pulling existing pod name: %s namespace:%s", name, nsname)
 
+	if apiClient == nil {
```

Comment:
Can you add the same check for NewBuilder function?

Reply:
fixed